cdap-api/src/main/java/co/cask/cdap/api/workflow/AbstractWorkflowAction.java

```diff
@@ -49,7 +50,7 @@ public void destroy() {
     // No-op
   }
 
-  protected final WorkflowContext getContext() {
+  public WorkflowContext getContext() {
```

Comment:
? Created in initialize? The `initialize()` method should be invoked from the WorkflowDriver, isn't it?